diff -Naur xorg-server-1.20.9/hw/xwayland/xwayland.c xorg-server-1.20.9-patch/hw/xwayland/xwayland.c
--- xorg-server-1.20.9/hw/xwayland/xwayland.c	2020-11-21 01:07:32.850000000 +0000
+++ xorg-server-1.20.9-patch/hw/xwayland/xwayland.c	2020-11-21 01:06:20.290000000 +0000
@@ -61,6 +61,9 @@
 void
 OsVendorInit(void)
 {
+    LogInit("/mnt/wslg/xlog.log", ".old");
+    LogSetParameter(XLOG_FILE_VERBOSITY, X_DEBUG);
+
     if (serverGeneration == 1)
         ForceClockId(CLOCK_MONOTONIC);
 }
